src/pages/migrating/guide/develop.mdx

```diff
@@ -10,613 +10,315 @@ import { Tag } from '@carbon/react';
-    <Grid>
-      <Column sm={1} md={1} lg={2}>
-        <p>sm={1}</p> <p>md={1}</p> <p>lg={2}</p>
-      </Column>
-      <Column sm={1} md={1} lg={2}>
-        <p>sm={1}</p> <p>md={1}</p> <p>lg={2}</p>
-      </Column>
-      <Column sm={0} md={1} lg={1}>
-        <p>sm={0}</p> <p>md={1}</p> <p>lg={1}</p>
-      </Column>
-      <Column sm={0} md={1} lg={1}>
-        <p>sm={0}</p> <p>md={1}</p> <p>lg={1}</p>
-      </Column>
-      <Column sm={0} md={0} lg={1}>
-        <p>sm={0}</p> <p>md={0}</p> <p>lg={1}</p>
-      </Column>
-      <Column sm={0} md={0} lg={1}>
-        <p>sm={0}</p> <p>md={0}</p> <p>lg={1}</p>
-      </Column>
-      <Column sm={0} md={0} lg={1}>
-        <p>sm={0}</p> <p>md={0}</p> <p>lg={1}</p>
-      </Column>
-      <Column sm={0} md={0} lg={1}>
-        <p>sm={0}</p> <p>md={0}</p> <p>lg={1}</p>
-      </Column>
-    </Grid>
-  </Column>
-  <Column sm={0} md={0} lg={3}>
-    <p>Small: Span 0 of 4</p>
-    <p>Medium: Span 0 of 8</p>
-    <p>Large: Span 3 of 16</p>
-  </Column>
-</Grid>
-```
-
-### Resources
-
-<Row className="resource-card-group">
-  <Column colLg={4} colMd={4} noGutterSm>
-    <ResourceCard
-      subTitle="CSS Grid docs"
-      href="https://github.com/carbon-design-system/carbon/blob/main/docs/migration/11.x-grid.md"
-    >
-      <MdxIcon name="github" />
-    </ResourceCard>
-  </Column>
-  <Column colLg={4} colMd={4} noGutterSm>
-    <ResourceCard
-      subTitle="CSS Grid storybook"
-      href="https://github.com/carbon-design-system/carbon/blob/main/docs/migration/11.x-grid.md"
-    >
-      <MdxIcon name="react" />
-    </ResourceCard>
-  </Column>
-</Row>
-
-<!-- to-do: link to live v11 storybook once we have it up!!!! -->
-
-## Theming
-
-With our theming updates we have some new features and some breaking changes.
-You can now specify you're theme with our new
-[Theme component](#theme-component)., and incorporate inline theming. You can
-also specify a layer with a [Layer component](#layer-component).
-
-### Layer Component <Tag type="cyan">Breaking</Tag>
-
-In regards to theming, we have also added in a new `Layer` component. The
-`Layer` component replaces the `light` prop. It is a container that renders the
-`cds--layer` class name. The class name then remaps contextual tokens to the
-next layer. This allows you to style different layers, or directly use the
-tokens exported from our styles package.
-
-#### What this means for you and your team
-
-- Layer componet replaces `light` prop. See the first example below.
-- More flexibility for 3 layer stacking vs. 2 layer stacking
-- Third layer stacking will now be supported in dark themes
+```scss
+@use '@carbon/react' with ($css--default-type: true, $css--reset: true);
+```
 
-```jsx
-  // v10
-  <Dropdown light />
+If you were using any feature flags in v10, you can safely remove them in v11.
 
-  // v11
-  <Layer>
-    <Dropdown />
-  </Layer>
-```
+_Note: you can also use `@import` to bring in Carbon, if you prefer, although
```

Comment:
On the website we don't normally do notes like this in italic, we normally just use the inline notification component. Its probably fine to leave it as it though.

Reply:
@aagonzales great point. One thing I noticed was that I wanted to distinguish between "strong suggestions" where we really want to recommend this thing and "passive suggestions" which were kind of throwaway for completeness.

Would you recommend I group both of these under inline notifications so that it's consistent with the rest of the site? Happy to make the change, just wasn't sure what the convention was

Reply:
Yeah I think that distinction is fine. I talked to Lauren about it and we think the format just needs change. The italic was just feeling a little out of place and we think this format could work better

Reply:
Sounds good, done!